pkg/file/builder.go

```diff
@@ -681,7 +682,7 @@ func (b *stateBuilder) consumers() {
 		}
 
 		for _, c := range consumers {
-			err = b.intermediate.Consumers.Add(*c)
+			err = b.intermediate.Consumers.AddIgnoringDuplicates(*c)
```

Comment:
Are consumers also preprocessed while processing a consumer-group (similar to routes nested under services)?